packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 50.14.0 - Month DD, YYYY
```

Comment:
```suggestion
# 50.14.0 - January 20, 2021
```

Reply:
Moratorium ends midnight tonight. We will release tomorrow, so changed date to January 21 instead.